pkg/rpc/server/server.go

```diff
@@ -463,13 +463,23 @@ func (s *Server) getPeers(_ request.Params) (interface{}, *response.Error) {
 	return peers, nil
 }
 
-func (s *Server) getRawMempool(_ request.Params) (interface{}, *response.Error) {
+func (s *Server) getRawMempool(reqParams request.Params) (interface{}, *response.Error) {
+	var getVerified bool
+	if len(reqParams) > 0 {
+		getVerified = reqParams.Value(0).GetBoolean()
+	}
```

Comment:
There is no need to check parameters count: `getVerified := reqParams.Value(0).GetBoolean()`